### PR TITLE
DEVDOCS-3076 - update Next.js Commerce guide

### PR DESCRIPTION
**Pull request by markcmurphy**
# [DEVDOCS-3076](https://jira.bigcommerce.com/browse/DEVDOCS-3076)

## What changed?
* update images and dialogue text to reflect updated Vercel signup process
* add updated `.env.template` fields and info on values

### CODE REVIEW

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 1]
```diff
@@ -51,19 +51,21 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
...
 
-   ![Log in with existing Vercel account](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-03.png)
+3. After signing in, a **Create Git Repository** dialogue will appear with a dropdown to select your Git Scope, and a **Repository Name** text field. You may change the name of the project from the default or leave it as-is. Click **Create** to proceed.
 
-3. After signing in, a **Clone Git Repository** dialogue will appear with a link to the Next.js Commerce repository, a dropdown to select your Vercel account, and a **Project Name** text field. You may change the name of the project from the default or leave it as-is. Click **Continue** to proceed.
+   ![Create Git repository](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_repo.png)
 
-   ![Clone Git repository](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-04.png)
+4. The next dialogue asks if you would like to create a team or use your personal account. You may click on **Skip** if you don't want to create a Vercel Team.
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:60]:
```suggestion
4. (Optional) Create a team or select your personal account. You can also click **Skip** to continue without creating a Vercel Team.
```

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 2]
```diff
@@ -51,19 +51,21 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
...
 
-4. In the **Install Integrations** dialogue box, click **Install** next to the BigCommerce integration to connect your BigCommerce store to your Vercel project.
+   ![Create Vercel team](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_team.png)
 
-   ![Install BigCommerce integration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-05.png)
+5. In the **Add Integrations** dialogue box, click **Add** next to the BigCommerce integration to connect your BigCommerce store to your Vercel project.
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:64]:
```suggestion
5. In the **Add Integrations** dialogue box, click **Add** to connect your BigCommerce store to your Vercel project.
```
Google developer documentation style guide advises against using directional language to orient readers.

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 3]
```diff
@@ -51,19 +51,21 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
...
 
-5. The **Add BigCommerce to Your Vercel Project** dialogue will have options to either **Sign Up** or **Log In**.
+   ![Add BigCommerce integration](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_add_integrations.png)
+
+6. The **Add BigCommerce to Your Vercel Project** dialogue will have options to either **Sign Up** or **Log In**.
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:68]:
```suggestion
6. In the **Add BigCommerce to Your Vercel Project** dialogue, choose **Sign Up** or **Log In**.
```

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 4]
```diff
@@ -77,23 +79,17 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
 
       ![Sign up for BigCommerce store registration form](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-09.png)
 
-6. When the integration completes, you will see a blue checkmark where the **Install** button used to be. Click **Continue**.
-
-   ![BigCommerce integration installation confirmation](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-10.png)
-
-7. The **Create Git Repository** dialogue box will display a message about importing the `main` branch of the Next.js Commerce repository into your Vercel account. To complete the process, select your Git provider and name the repository where you want Next.js Commerce cloned and deployed to Vercel. Click **Continue**.
+7. When the integration completes, you will see that the **Add** button has turned to a **Change** button.
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:82]:
```suggestion
7. When the integration completes, you will see a **Change** button.
```

Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:82]:
```suggestion
7. When the integration completes, the **Add** button will be replaced by a **Change** button.
```
Maybe there is a better way to put it?

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 5]
```diff
@@ -77,23 +79,17 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
...
 
-   ![Create Git repository dialogue](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-11.png)
+   ![BigCommerce integration installation confirmation](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_integrations_added.png)
 
-   ![Create Git repository continue button](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-12.png)
+8. Next.js Commerce will now begin to deploy
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:86]: Same here. I think we can remove this step.

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 6]
```diff
@@ -108,12 +104,25 @@ BIGCOMMERCE_STORE_API_TOKEN=${STORE_TOKEN}
...
+BIGCOMMERCE_STOREFRONT_API_TOKEN=${STOREFRONT_API_TOKEN}
+BIGCOMMERCE_STORE_API_URL=https://api.bigcommerce.com/stores/${STORE_ID}
+BIGCOMMERCE_STORE_API_TOKEN=${STORE_TOKEN}
+BIGCOMMERCE_STORE_API_CLIENT_ID=${STORE_CLIENT}
+BIGCOMMERCE_CHANNEL_ID=${CHANNEL_ID}
+BIGCOMMERCE_STORE_URL=https://store-${STORE_HASH}.mybigcommerce.com
+BIGCOMMERCE_STORE_API_STORE_HASH=${STORE_HASH}
+```
+
+- `STORE_HASH`: Your store hash can be retrieved from your BigCommerce Store Control Panel URL in the format of `https://store-${STORE_HASH}.mybigcommerce.com/manage/dashboard`
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:119]:
```suggestion
- `STORE_HASH`: You can retrieve it from your BigCommerce store control panel URL in the format of `https://store-${STORE_HASH}.mybigcommerce.com/manage/dashboard`
```

File: docs/api-docs/building-storefronts/nextjs-commerce-guide.md [hunk 7]
```diff
@@ -108,12 +104,25 @@ BIGCOMMERCE_STORE_API_TOKEN=${STORE_TOKEN}
...
 
-  ![STORE-ID location in URL](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-15.png)
+  ![STORE-HASH location in URL](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-15.png)
 
 - `STORE_TOKEN|STORE_CLIENT`: For instructions on generating Store API credentials, visit [Obtaining Store API Credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
-- `STOREFRONT_TOKEN`: For instructions on generating the Storefront API token, visit [Create a Token](https://developer.bigcommerce.com/api-reference/store-management/tokens/api-token/createtoken).
+- `STOREFRONT_API_TOKEN`: For instructions on generating the Storefront API token, visit [Create a Token](https://developer.bigcommerce.com/api-reference/store-management/tokens/api-token/createtoken).
+- `CHANNEL_ID`: Use the channel ID of the channel created by Vercel during deployment. For information on how to retrieve channel information, visit [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels)
```
Comment (ektravel) [inline docs/api-docs/building-storefronts/nextjs-commerce-guide.md:125]:
```suggestion
- `CHANNEL_ID`: Use the channel ID of the channel created by Vercel during deployment. For information on how to retrieve channel information, visit [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels).
```

**Review by ektravel — COMMENTED** [review]
Added some suggestions.

**Comment by markcmurphy** [comment]
@uwikaiddi, this PR is missing a team member's approval, but I implemented all Katya's feedback, so it should be good to go.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**